docs/3.x.x/api-reference/reference.md

```diff
@@ -27,6 +28,11 @@
 
 This object contains the controllers, models, services and configurations contained in the `./admin` folder.
 
+## strapi.api
+
+This object contains APIs contained in the `./api` folder.
+And by using `strapi.api[:name]` this object contains the controllers, models, services and configurations contained in the `./api/:name` folder.
```

Comment:
And by using `strapi.api[:name]` you can access the controllers, services, the model definition and also the configurations of the `./api/:name` folder